### PR TITLE
Merge `9.0.x` into `develop`

### PR DESCRIPTION
**Pull request by nicosomb**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Merge `9.0.x` into `develop`
| Type?             | improvement
| Category?         | ME
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI & UI Tests 🟢
| UI Tests          | https://github.com/nicosomb/ga.tests.ui.pr/actions/runs/12354960390
| Sponsor company   | PrestaShop SA

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]
Ok for UI Tests

**Review by jolelievre — DISMISSED** [review]

**Review by boherm — DISMISSED** [review]

File: tests/UI/package-lock.json [hunk 1]
```diff
@@ -432,7 +432,12 @@
     },
     "node_modules/@prestashop-core/ui-testing": {
       "version": "0.0.12",
+<<<<<<< HEAD
       "resolved": "git+ssh://git@github.com/PrestaShop/ui-testing-library.git#87beff248b2aa6d520c2e54057594a5e594a3911",
+=======
+      "resolved": "git+ssh://git@github.com/PrestaShop/ui-testing-library.git#f89ac228b2a7c6a82bef5f291a8b0216b2ab12d4",
+      "license": "MIT",
+>>>>>>> 9.0.x
```
Comment (Progi1984) [inline tests/UI/package-lock.json]:
```suggestion
      "resolved": "git+ssh://git@github.com/PrestaShop/ui-testing-library.git#f89ac228b2a7c6a82bef5f291a8b0216b2ab12d4",
      "license": "MIT",
```

File: tests/UI/package-lock.json [hunk 2]
```diff
@@ -8444,7 +8449,11 @@
       }
     },
     "@prestashop-core/ui-testing": {
+<<<<<<< HEAD
       "version": "git+ssh://git@github.com/PrestaShop/ui-testing-library.git#87beff248b2aa6d520c2e54057594a5e594a3911",
+=======
+      "version": "git+ssh://git@github.com/PrestaShop/ui-testing-library.git#f89ac228b2a7c6a82bef5f291a8b0216b2ab12d4",
+>>>>>>> 9.0.x
```
Comment (Progi1984) [inline tests/UI/package-lock.json]:
```suggestion
      "version": "git+ssh://git@github.com/PrestaShop/ui-testing-library.git#f89ac228b2a7c6a82bef5f291a8b0216b2ab12d4",
```

**Review by Progi1984 — CHANGES_REQUESTED** [review]
@nicosomb Fix to `tests/UI/package-lock.json`

**Review by Progi1984 — APPROVED** [review]
ok for ui tests

**Review by boherm — APPROVED** [review]

**Pull request merged**